dotcom-rendering/src/components/SignInGateSelector.importable.tsx

```diff
@@ -476,6 +486,11 @@ const decideDailyArticleCount = (): number => {
 
 const decideAuxiaProxyReaderPersonalData =
 	async (): Promise<AuxiaGateReaderPersonalData> => {
+		return Promise.resolve({
```

Comment:
remove?!

Reply:
It's gone (you find it between two commits)